### DISCUSSION TITLE
The lists in Github may works without a star action.

### DISCUSSION THREAD

**Discussion by juniobiel** [community/community#125708] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi all. I want to give a personal feedback to a functionality that I think is very usefull to use, but needs some adjustments.

I've tryed to use the Lists in Github to categorize my repositories or others. But to use the functionality I need to put these repositories as favorite/starred.
So, it's breaks my usability in some ways.

I think that is very better when I want to add some repository to a list, I can just add. Without link it with a star.
Maybe, the starrred repositories needs to be it, favorites, fast thing.
I use my favorite/star to repositories that I see some usability or interesting thing or just liked it.

**Comment by Raunaksplanet** [comment]
Can you please explain how its breaking your usability ?

**Reply by juniobiel** [reply]
Yeah man, it's simple.

I want to add categorization to my repositories without starring them, just adding them to a list. And I like to use stars for other repositories that I'm not the author of, do you know?

I think that it can be more usefull.

**Comment by diyardev** [comment]
What? I don't understand what you mean?

31 May 2024 Cum 06:03 tarihinde Raunak Gupta ***@***.***>
şunu yazdı:

**Comment by samuelralmeida** [comment]
I agree with juniobiel. Today, when you add a repository to a list, that repository also gets a star. However, I would like the list feature to allow grouping repositories without giving them stars. I think it could work similarly to labels in Gmail.

**Comment by danialtavakoli** [comment]
It sounds like you find the Lists feature in GitHub useful but feel it could be improved by allowing repositories to be added to a list without needing to star them. You’d prefer to keep starred repositories as your favorites or those you find interesting, separate from the lists you create for categorization. Your suggestion is that the Lists feature should allow direct addition of repositories without linking them to the star functionality. This would enhance usability by letting you organize repositories in lists independently of your favorites.

**Comment by Azem92** [comment]
[]()[]()

**Comment by jazyokey** [comment]
Login

**Comment by 532910** [comment]
I second that! Star is a like (❤️), it's a positive assessment, it means I like this repo. It has nothing common with a list, that is a collection. I would not like to star repo, but would like to remember it, and the current implementation of the lists is competently unusable for this, so I have to bookmark them in browser.

**Comment by SantiagoRR2004** [comment]
I also agree with the idea of separating lists from stars. This topic also appears in https://github.com/orgs/community/discussions/8842 and https://github.com/orgs/community/discussions/9067.
